duckduckgo/iOS
# Accessibility improvements

Task/Issue URL: https://github.com/duckduckgo/iOS/issues/1241
Tech Design URL:
CC:

**Description**:
Improves accessibility access
Fixes text color with Dark Theme in Autofill -> item Details



**Steps to test this PR**:
1. Xcode -> Developer tools -> Accessibility Inspector - OR - Reveal app (has trial period)
2. Validate added labels are working and fix the VoiceOver issues

+ Find In Page
  + When Find In Page toolbar displayed, main navigation toolbar with buttons is still present in VoiceOver and they overlap
  + Search Field has "Text Field, Is Editing" name, should be something like find in page field
  + "find previous dark"/"find next dark" (image names) buttons should be "Find Next" "Find Previous" buttons
  + 1 of 3 (results) should go before "Done" in navigation order

+ Report Broken Site
  + describe what happened items should be "Button"-s (now Text)

+ Search suggestions
  + narrate suggestion image (website/search for/history) for a suggested item

+ Downloads
  + Items have no default action (should turn file name into a button - open file)
  + Share button has "DownloadsShareIcon, Button" title should be "Share"
  + in Edit mode "Delete item" icon is not present for VoiceOver

+ Settings
  + Set as Default Browser is a "Static text" instead of "Button"
  + Themes: selectable items are "Static Text" instead of "Button"
  + Fire Button Animation: selectable items are "Static Text" instead of "Button"
  + Text Size: Adjustable Control shows "22% adjustable" = "Text size - 100%"
  + Fireproof Sites: Edit Mode: Remove All Fireproof Sites is "Static Text" instead of "Button"
  + Automatically Clear Data: selectable items are "Static text" instead of "Button"
  + About: DuckDuckGo logo should be narrated as "DuckDuckGo" (now only "Privacy, simplified")
  + Block email protection: link when no link?
  + GPC: Learn more link?

+ Autofill
  + Login details: image shouldn‘t be narrated
  + Item details -> tap cell for copy menu to appear -> choose Edit - crash



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
